playwright/parser.go

```
@@ -0,0 +1,81 @@
+type TestRepo struct {
+	Suites []TestSuite
+}
+
+// Ex: [5] - A 5 pt private test
+// Ex: [10*] - A 10 pt public test
+func ParseTestName(name string) (TestMeta, error) {
+	re := regexp.MustCompile(`\[(\d+)(\*)?\] \- (.*)`)
+	matches := re.FindStringSubmatch(name)
+
+	if len(matches) != 4 {
+		return TestMeta{}, errors.New("Did not match test name regex")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)